ts/runtime.ts

```diff
@@ -284,6 +284,35 @@ export class ElementaryTestingError extends Error {
     super(message);
   }
 }
+/**
+ * Returns the default JSON object with the parse function modified to
+ * return a 'stopified' array
+ *
+ * @returns modified JSON object
+ */
+function getStopifiedJSON() {
+  let jsonStopified = Object.create(JSON);
+  jsonStopified.parse = (text: string) => stopifyObjectArrayRecur(JSON.parse(text));
+  return jsonStopified;
+}
+
+export const JSONStopified = getStopifiedJSON();
+
+/**
+ * Returns the default Object object with few functions modified to
+ * return a stopified object.
+ * 
+ * @return modified Object object
+ */
+function getStopifiedObject() {
+  let objectStopified = Object.create(Object);
+  objectStopified.keys = (obj : any) => stopifyArray(Object.keys(obj));
+  objectStopified.values = (obj: any) => stopifyArray(Object.values(obj));
+  objectStopified.entries = (obj: any) => stopifyArray(Object.entries(obj));
+  return objectStopified
+}
+
+export const ObjectStopified = getStopifiedObject();
```

Comment:
So, this is giving students access to `Object.keys`, `Object.values`, etc. Do they need it for HW4?

Reply:
Hmm, no, students should not be using them. They should use `lib220.getProperty`.

Reply:
OK. I don't actually understand why this commit is needed. Yes, we need to export Object to support classes. But, why are we bothering with calling stopifyArray above?